.golangci.yml

```diff
@@ -1,46 +1,152 @@
+    - deadcode
+    - depguard
+    - dupl
+    - exhaustive
+    - funlen
+    - gci
+    - gochecknoglobals
+    - gochecknoinits
+    - gocognit
+    - goconst
+    - gocritic
+    - gocyclo
+    - godot
+    - godox
+    - goerr113
+    - gofmt
+    - gofumpt
+    - goheader
+    - goimports
+    - golint
+    - gomnd
+    - gomodguard
+    - gosec
+    - ifshort
+    - interfacer
+    - lll
+    - makezero
+    - maligned
+    - megacheck
+    - misspell
+    - nestif
+    - nlreturn
+    - noctx
+    - nolintlint
+    - paralleltest
+    - prealloc
+    - rowserrcheck
+    - scopelint
+    - structcheck
+    - stylecheck
+    - testpackage
+    - thelper
+    - tparallel
+    - wastedassign
+    - whitespace
+    - wrapcheck
+    - wsl
 
 linters-settings:
   revive:
     rules:
+      - name: argument-limit
+        arguments: [ 6 ]
+      - name: atomic
+      - name: bare-return
       - name: blank-imports
+      - name: bool-literal-in-expr
+      - name: call-to-gc
+      - name: confusing-naming
+      - name: confusing-results
+      - name: constant-logical-expr
       - name: context-as-argument
       - name: context-keys-type
+      - name: deep-exit
+      - name: defer
       - name: dot-imports
+      - name: duplicated-imports
+      - name: early-return
+      - name: empty-block
+      - name: empty-lines
+      - name: error-naming
       - name: error-return
       - name: error-strings
-      - name: error-naming
-      - name: exported
+      - name: errorf
+      - name: flag-parameter
+      - name: function-result-limit
+        arguments: [ 3 ]
+      - name: identical-branches
       - name: if-return
+      - name: imports-blacklist
+        arguments: [ "log" ]
+      - name: import-shadowing
       - name: increment-decrement
-      - name: var-naming
-      - name: var-declaration
+      - name: indent-error-flow
+      - name: modifies-parameter
+      - name: modifies-value-receiver
       - name: package-comments
       - name: range
+      - name: range-val-address
+      - name: range-val-in-closure
       - name: receiver-naming
-      - name: time-naming
-      - name: unexported-return
-      - name: indent-error-flow
-      - name: errorf
-      - name: empty-block
+      - name: redefines-builtin-id
+      - name: string-of-int
+      - name: struct-tag
       - name: superfluous-else
-      - name: unused-parameter
+      - name: time-naming
+      - name: unconditional-recursion
+      - name: unexported-naming
+      - name: unhandled-error
+      - name: unnecessary-stmt
       - name: unreachable-code
-      - name: redefines-builtin-id
+      - name: unused-parameter
+      - name: var-declaration
+      - name: var-naming
+      - name: waitgroup-by-value
+  nakedret:
+    # make an issue if func has more lines of code than this setting and it has naked returns; default is 30
+    max-func-lines: 1
 
 run:
+  # exit code when at least one issue was found, default is 1
+  issues-exit-code: 0
```

Comment:
This can be removed, it is handled in the github action config.

Reply:
Removed